Week1/Section3/Exercise2/Source2.cpp

```diff
@@ -53,39 +59,45 @@ bool makeQuestion(std::string Question, bool CorrectAnswer)
 
 int main()
 {
-	int UserCorrectAnswers = 0;
-	float MaxCorrectAnswers = 3;
-
 	// stores the Questions and the correct Answers
 	std::string Questions[3] = {
 		"Is the ocean a soup? ",
 		"Is the forest a salad? ",
-		"Is a hot dog a taco though? "};
+		"Is a hot dog a taco though? "
+	};
 
 	bool Answers[3] = {
 		true,
 		false,
-		false};
+		false
+	};
 
-	std::cout << "Welcome to your quiz!\nAnswer\nF for False\nT for True\nYou only have 3 attempts per cuestion.\n"
-			  << std::endl;
+	std::cout << "Welcome to your quiz!\nAnswer\nF for False\nT for True\nYou only have 3 attempts per cuestion.\n" << std::endl;
 
+	bool bQuizAproved;
```

Comment:
This logic is perfectly fine 👍 However, I'd like to point out that in this case, it may be clearer to set the default value to true and then only set the value to false and break the loop when the answer is wrong. This is because bQuizApproved is being set as true in every iteration of the loop, which is a bit unnecessary and misleading.

This case is different than the previous cases I commented on because the logic here is "this is true until proven to be false", which makes sense, while the logic on earlier versions of the exercises was to put an arbitrary value just to make sure the first condition is false.

Reply:
Did you see this comment? It doesn't have a reply and isn't addressed hehe 😅

Reply:
hmmm :/ sorry I'm not quite sure if understand this issue, what I see is that you are asking for a more understandable condition for the loop?

I tried to simplify the code and add an explanatory comment 😵